lib/datapacksutils.js

```diff
@@ -3042,8 +3042,90 @@ DataPacksUtils.prototype.loadSalesforceMetadata = async function() {
 
 var folderOnlyMetadata = [ "aura", "layouts", "lwc", "staticresources" ]
 
-DataPacksUtils.prototype.retrieveSalesforce = async function(jobInfo, listOnly) {
+DataPacksUtils.prototype.getAdditionalSFMetadataDependencies = async function(jobInfo){
+    let vlocityDataPackTypes = [],
+        salesforceKeys = [],
+        salesforceTypes = [];
+
+    Object.keys(jobInfo.queries).forEach(key => {
```

Comment:
This is not really an accurate way to get the Vlocity vs Salesforce types. Would probably be better to write a utility method that gets all the actual supported types from the Org in the VlocityDataPackConfiguration__mdt object depending on why this needs to happen.